plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/provider/NamespacedPropertyEditorDialog.java

```diff
@@ -454,6 +912,355 @@ public void handleEvent(Event event) {
+            // The following two methods were left empty intentionally since they must be overridden
+            @Override
+            public void mouseUp(MouseEvent e) {}
+
+            @Override
+            public void mouseDown(MouseEvent e) {}
+
+        });
+        
+        inlineRadioButton.addSelectionListener(new SelectionAdapter()  {
+            
+            @Override
+            public void widgetSelected(SelectionEvent e) {
+                Button source = (Button) e.getSource();
+                if(source.getSelection())  {
+                    filePathTextField.setText(EMPTY_STRING);
+                    xPathTextField.setText(EMPTY_STRING);
+                    outputXMLTextArea.setText(EMPTY_STRING);
+                    inlineXMLTextArea.setText(XML_CONTENT_PLACEHOLDER_TEXT);
+                    infoLabel.setText(INFO_LABEL_DEFAULT_TEXT);
+                    
+                    browseButton.setEnabled(false);
+                    filePathTextField.setEnabled(false);
+                    
+                    xmlContentTabFolder.setSelection(0);
+                    xmlContentTabFolder.getTabList()[0].setEnabled(true);
+               
+                    treeViewWidget.removeAll();
+                }
+            }
+        });
+         
+        browseFileRadioButton.addSelectionListener(new SelectionAdapter()  {
+ 
+            @Override
+            public void widgetSelected(SelectionEvent e) {
+                Button source = (Button) e.getSource();
+                if(source.getSelection())  {
+                    browseButton.setEnabled(true);
+                    infoLabel.setText(INFO_LABEL_DEFAULT_TEXT);
+                    
+                    xmlContentTabFolder.setSelection(1);
+                    treeViewWidget.removeAll();
+                    inlineXMLTextArea.setText(EMPTY_STRING);
+                    outputXMLTextArea.setText(EMPTY_STRING);
+                }
+            } 
+        });
+        
+        inlineXMLTextArea.addLineStyleListener(new LineStyleListener()
+        {
+            public void lineGetStyle(LineStyleEvent e)
+            {
+                e.bulletIndex = inlineXMLTextArea.getLineAtOffset(e.lineOffset);
+
+                StyleRange style = new StyleRange();
+                Device device = Display.getCurrent();
+                final RGB LINE_NUMBER_BG = new RGB(0, 0, 100);
+                final RGB LINE_NUMBER_FG = new RGB(255, 255, 255);
+                
+                style.background = new Color(device, LINE_NUMBER_BG);
+                style.foreground = new Color(device, LINE_NUMBER_FG);
+                
+                style.metrics = new GlyphMetrics(0, 0, Integer.toString(inlineXMLTextArea.
+                        getLineCount() + 1).length() * 12);
+
+                e.bullet = new Bullet(ST.BULLET_NUMBER, style);
+            }
+        });
+        
+        inlineXMLTextArea.addFocusListener(new org.eclipse.swt.events.FocusListener() {
```

Comment:
Import the package.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804